lib/caqti_connection_sig.mli

```diff
@@ -135,14 +135,16 @@ module type S = sig
     ('b -> 'c -> 'c) ->
     'a -> 'c -> ('c, [> Caqti_error.call_or_retrieve] as 'e) result future
   (** Combining {!call} with {!Response.fold}, this sends a request to the
-      database and folds over the result rows. *)
+      database and folds over the result rows in a single blocking operation.
+  *)
 
   val fold_s :
     ('a, 'b, [< `Zero | `One | `Many]) Caqti_request.t ->
     ('b -> 'c -> ('c, 'e) result future) ->
     'a -> 'c -> ('c, [> Caqti_error.call_or_retrieve] as 'e) result future
   (** Combining {!call} with {!Response.fold_s}, this sends a request to the
-      database and folds concurrently over the result rows.
+      database and folds sequentially over the result rows in a non-blocking
+      manner.
```

Comment:
Yes, it's good to mention "sequentially" here, and maybe it's then easier to read when also replacing "concurrently" with "non-blocking" to avoid the double adverb.

Reply:
I'm not sure I understand whether you're happy with this change or you're suggesting a change, in which case I'm of course happy to improve it!

Reply:
Your reformulation is good, I was thinking out loudly.

Reply:
great!